File: library/src/main/java/cn/wandersnail/widget/listener/RejectableViewClickCallback.java
package cn.wandersnail.widget.listener;

import android.view.View;

/**
 * date: 2019/8/22 16:43
 * author: zengfansheng
 */
public interface RejectableViewClickCallback {
    default void onReject(View view) {}

    void onAccept(View view);
}

File: library/src/main/java/com/snail/widget/listener/RejectableViewClickCallback.kt
package com.snail.widget.listener

import android.view.View

/**
 * 可以控制是否执行点击事件回调
 *
 * date: 2019-06-16 11:55
 * author: zengfansheng
 */
interface RejectableViewClickCallback {
    fun onReject(v: View)

    fun onAccept(v: View)
}